.expeditor/publish.pipeline.yml

```diff
@@ -2,11 +2,50 @@ expeditor:
   secrets: true
 
 steps:
-- label: List versions
-  commands:
-    - make list.versions
-  env:
-    AZURE_CLOUD: public
+- block: "Internal Publish"
+  prompt: "Fill out the details for release"
+  fields:
+    - select: "Platform"
+      key: "azure-platform"
+      options:
+        - label: "Ubuntu"
+          value: "ubuntu"
+        - label: "Windows"
+          value: "windows"
+    - select: "Cloud"
+      key: "azure-cloud"
+      options:
+        - label: "Public"
+          value: "public"
+        - label: "Government"
+          value: "government"
+    - select: "Confirmation"
+      key: "azure-confirmation"
+      default: "true"
+      options:
+        - label: "True"
+          value: "true"
+        - label: "False"
+          value: "false"
+    - text: "Release Version"
+      key: "release-version"
+      hint: "121X.XX.X.X"
+    - text: "Region 1"
+      key: "deploy-region1"
+      hint: "Which first region should we deploy this to? Please don't enter region in double-quotes,eg.,\"East US\""
+      required: false
+    - text: "Region 2"
+      key: "deploy-region2"
+      hint: "Which second region should we deploy this to? Please don't enter regions in double-quotes,eg.,\"East US\""
+      required: false
+    - select: "Confirme deployment type"
+      key: "deployment-type"
+      default: "confirm_internal_deployment "
+      options:
+        - label: "Internal Deployment"
+          value: "confirm_internal_deployment "   
+- command: /workdir/scripts/internal-publish.sh
+  label: ":rocket:"
```

Comment:
In yml you don't need to enclose text values in quotes. Pls remove quotes.

Reply:
We are required to put the attribute values in quotes for the block/input steps, labels etc. I haven't used quotes where it's not required. eg., for **command**  step. You may refer to buildkite docs for block step [here](https://buildkite.com/docs/pipelines/block-step).